### PR TITLE
Fixing issues with status code ranges

### PR DESCRIPTION
**Pull request by andrewhdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Cloud IoT API
+
+[![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/iot.svg)](https://pkg.go.dev/cloud.google.com/go/iot)
```
Comment (clarkbynum) [inline README.md:3]: Just to be sure - we're still planning on uploading our own docs, right? I believe there's a jira for that

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Go Client Library for ClearBlade IoT Core API.
+
+## Authorization
+
+See the [Authorization](https://clearblade.atlassian.net/wiki/spaces/IC/pages/2240675843/Add+service+accounts+to+a+project)
+documentation for authorizing sdk with clearblade iot-core
+
+## API Documentation
+
+View the Go secionts of these [Samples](https://clearblade.atlassian.net/wiki/spaces/IC/pages/2310897676/Samples)
```
Comment (clarkbynum) [inline README.md:14]: Typo `secionts`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Cloud IoT API
+
+[![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/iot.svg)](https://pkg.go.dev/cloud.google.com/go/iot)
+
+Go Client Library for ClearBlade IoT Core API.
+
+## Authorization
+
+See the [Authorization](https://clearblade.atlassian.net/wiki/spaces/IC/pages/2240675843/Add+service+accounts+to+a+project)
```
Comment (clarkbynum) [inline README.md:9]: Are there any docs for the CLEARBLADE_CONFIGURATION env variable? Here's how we call it out in the node readme - https://github.com/ClearBlade/nodejs-iot/#setting-up-service-account-credentials

**Review by clarkbynum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andrewhdev) [reply README.md:3]: Yes, I'll remove that section for now. I'll change this to link to our docs.

**Review by clarkbynum — APPROVED** [review]

**Pull request merged**